config.json

```diff
@@ -444,6 +444,50 @@
     ]
   },
   "concepts": [],
-  "key_features": [],
-  "tags": []
+  "key_features": [
+    {
+      "icon": "features-oop",
+      "title": "Domain Specific",
+      "content": "R is a language and environment for statistical computing and graphics."
+    },
+    {
+      "icon": "features-strongly-typed",
+      "title": "Cross-platform",
+      "content": "It is free, open source, and available on every major platform."
+    },
+    {
+      "icon": "features-functional",
+      "title": "Multi-paradigm",
+      "content": "R has a functional programming foundation, but facilitates procedural or object-oriented approaches."
```

Comment:
Nice how you've managed to cap it at exactly 100 characters :)

Reply:
Haha yeah. I had already drafted these before I was aware of the character limits so trimmed some of the longer ones down using R to check the character counts as I did :-)